weave/evaluation/eval_imperative.py

```diff
@@ -769,3 +769,206 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
             "be removed in a future version."
         )
         super().__init__(*args, **kwargs)
+
+
+
+
+
+class LiteEvaluationLogger(EvaluationLogger):
+    """Simplified interface for logging evaluations with Weave.
+
+    Combines prediction and score logging into a single `log_example()` call.
+    Automatically calls `log_summary()` at program exit via atexit.
+
+    **Important:** All parameters (inputs, output, scores) must be dicts.
+
+    Args:
+        model: Model identifier string or Model instance.
+        dataset: Dataset identifier string, Dataset instance, or list of dicts.
+        eval_attributes: Optional dict of evaluation metadata.
+        scorers: Optional list of predefined scorer names.
+        name: Optional name for the evaluation call.
+
+    Example:
+        ```python
+        import weave
+        weave.init("my-project")
+
+        logger = LiteEvaluationLogger(model="my_model_v1", dataset="test_set")
+
+        logger.log_example(
+            inputs={"question": "What is 2+2?"},
+            output={"answer": "4"},
+            scores={"correct": True, "confidence": 0.95}
+        )
+
+        # Summary auto-logged at exit, or call manually 
+
+        ```
+    """
+
+    _done: bool = PrivateAttr(default=False)
+
+    def model_post_init(self, __context: Any) -> None:
+        """Initialize and register custom atexit handler."""
+        # Call parent initialization
+        super().model_post_init(__context)
+
+        # Remove from parent's global registry - we'll handle our own cleanup
+        if self in _active_evaluation_loggers:
+            _active_evaluation_loggers.remove(self)
+
+        # Register our own atexit handler
+        atexit.register(self._finalize_at_exit)
+
+    def _finalize_at_exit(self) -> None:
+        """Called automatically at program exit to finalize the evaluation."""
+        if self._done or self._is_finalized:
+            return
+        try:
+            self.log_summary()
+        except Exception:
+            logger.error("Error during automatic finalization at exit", exc_info=True)
+
+    def log_example(
```

Comment:
@andrewtruong would it make sense to define those APIs directly on the `EvaluationLogger` class instead of in a subclass? It would make it easier for users, no need to import a separate class.